src/Sulu/Bundle/AdminBundle/Resources/js/components/RectangleSelection/rectangleSelection.scss

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
Are you sure? This is done in ever scss file in the project so far. See `item.scss` in `Toolbar` for example.